pyproject.toml

```diff
@@ -1,5 +1,7 @@
 [project]
 requires-python = "~=3.13.3"
+name = "backend.ai"
+dynamic = ["version"]
 
```

Comment:
We’re managing internal components using `pants` in a monorepo.
In this setup, is it necessary to specify a name field in the root `pyproject.toml`?